extensions/eo/README.md

```diff
@@ -22,20 +22,28 @@ For defining view geometry of data, it is strongly recommended to use the [`view
   - [Landsat 8 with bands in Item Asset Definition and Collection Summaries](../item-assets/examples/example-landsat8.json)
 - [JSON Schema](json-schema/schema.json)
 
-## Item fields
+## Item Asset fields
 
 | Field Name     | Type                           | Description |
 | -------------- | ------------------------------ | ----------- |
-| eo:bands       | \[[Band Object](#band-object)] | This is a list of the available bands where each item is a [Band Object](#band-object). |
-| eo:cloud_cover | number                         | Estimate of cloud cover as a percentage (0-100) of the entire scene. If not available the field should not be provided. |
+| eo:bands       | \[[Band Object](#band-object)] | An array of available bands where each entity is a [Band Object](#band-object). |
 
-**eo:bands**: In previous versions `eo:bands` was allowed to be used on the asset-level referencing via array indices to the actual bands in Item `properties`. Starting with STAC 1.0.0-beta.1 you are now allowed to place the full `eo:bands` array with all Band Object information in Item `assets` as described in general in the [STAC Item](../../item-spec/item-spec.md#additional-fields-for-assets).
+**eo:bands**: In STAC versions 0.9.x and prior, `eo:bands` was only allowed to be used in an Asset by referencing the Band Object definitions in Item Properties via array index. Starting with STAC 1.0.0-beta.1, an Asset must have an `eo:bands` array containing Band Object entities, as described in general in the [STAC Item](../../item-spec/item-spec.md#additional-fields-for-assets).  
+
+## Item Properties fields
+
+| Field Name     | Type                           | Description |
+| -------------- | ------------------------------ | ----------- |
+| eo:cloud_cover | number                         | Estimate of cloud cover as a percentage (0-100) of the entire scene. If not available, the field should not be provided. |
+| eo:bands       | \[[Band Object](#band-object)] | An array of available bands where each entity is a [Band Object](#band-object). |
+
+**eo:bands**: This is the same as the attribute for Item Asset. As of STAC 1.0.0-beta.1, there is no way to reference these Band Objects from an Asset, so their primary use is simply to advertise a set of bands that may be available in some of the Assets in this Item. These can also be rolled up into Collection `summaries` object.
 
 ### Band Object
 
 | Field Name          | Type   | Description |
 | ------------------- | ------ | ----------- |
-| name                | string | The name of the band (e.g., "B01", "B02", "B1", "B5", "QA"). |
```

Comment:
a "QA" band doesn't conform to the definition of what these are supposed to be.  Might be useful in the future to have a "layer" extension or something similar for image "bands" that are derived values, e.g., a QA mask, NDVI, Land Surface Temerature

Reply:
I don't understand this. Why is QA not supposed to be a band if it's offered like the other bands?

Reply:
I had the same question, and I had previously been adding EO bands for QA (quality assurance) and derived data (e.g., NDVI, temperature). The intro to this extension "It
could consist of multiple spectral bands in any part of the electromagnetic spectrum. Examples of EO
data include sensors with visible, short-wave and mid-wave IR bands (e.g., the OLI instrument on
Landsat-8), long-wave IR bands (e.g. TIRS aboard Landsat-8)." -- which seems to me (apart from the "could" weasel word) to indicate it's only for data bands that represent wavelengths.

In the example for this extension, the "band" BQA doesn't have an eo:bands definition, whereas all the other data bands do have it. 

Also, since eo:bands have the wavelength and full_width_half_max, and all the common names are light spectra, it seems to be very specific to light from a sensor and not derived products like a QA band. 

I'm open to it either being specific to electromagnetic or any band/layer -- I was just trying to clarify the spec wrt to how I interpreted it.

Reply:
I see the point. I guss we should leave it open to implementors whether it's useful for them to add such bands or not. I could see good reasons to add them, I'd certainly do. But your arguments make sense, too.

Reply:
And thanks for clarifying!